specification/storage/resource-manager/Microsoft.Storage/stable/2021-06-01/blob.json

```diff
@@ -993,6 +993,14 @@
         "immutableStorageWithVersioning": {
           "$ref": "#/definitions/ImmutableStorageWithVersioning",
           "description": "The object level immutability property of the container. The property is immutable and can only be set to true at the container creation time. Existing containers must undergo a migration process."
+        },
```

Comment:
Please also update example

Reply:
It is part of existing published GA swagger